src/dashboard-button/dashboard-button.component.tsx

```diff
@@ -27,9 +35,10 @@ function nonSpaNavigate(event, url: string) {
   }
 }
 
-interface DashboardButtonProps extends RouteComponentProps {
+interface DashboardButtonProps {
```

Comment:
just out of curiosity, why did you use interface instead of type?

Reply:
Types cant extend but interfaces can. This was extending `RouteComponentProps`. Those, however, for whatever reason, introduce some meaningless TS errors. I got rid of the extend but left it as an interface.